src/frontend/src/lib/components/hero/Header.svelte

```diff
@@ -40,6 +41,7 @@
 		{/if}
 
 		{#if $authSignedIn}
+			<WalletConnectMenu />
```

Comment:
doe we keep the other WalletConnect button that is above this lines?? Or one exclude the other?

Reply:
You mean the `<WalletConnect/>`? This seems to be a wrapper around different WalletConnectModals, so yes I leave it.

Reply:
that button appears when an user is connected to WalletConnect; so if it's connected, we would have 2 wallet connect button?

Reply:
Ah you're completely right. Now I see what you meant